lunabot_embedded/firmware/teensy_main/lib/lunabot_drivers/RobotMsgs.proto

```diff
@@ -11,20 +11,17 @@ message RobotSensors {
   float drive_left_ang = 7;
   float drive_right_ang = 8;
   float dep_ang = 9;
-  float lead_screw_ang = 10;
-  float act_ang = 11;
```

Comment:
by deleting lead_screw/act angle sensor. the api for [actuation::update](https://github.com/PurdueLunabotics/purdue_lunabotics/blob/ad2dcf911504080b617f2cd0bc74a548211ee2c5/lunabot_embedded/firmware/teensy_main/teensy_main.ino#L33C1-L34C43) changes. need to make a change to actuation::update in robot.cpp and robot.h.

Reply:
done!